src/components/SlidingMenu.vue

```diff
@@ -104,8 +104,20 @@ export default {
       'getUserId',
       'getUserInfo',
       'getUserCivicLikerHalo',
+      'getUserIsCivicLiker',
+      'getUserIsCivicLikerTrial',
       'getHomeRoute',
     ]),
+
+    menuItemTitleForCivic() {
+      if (
+        (this.getUserId && !this.getUserIsCivicLiker) ||
+        this.getUserIsCivicLikerTrial
```

Comment:
`this.getUserId && (!this.getUserIsCivicLiker || this.getUserIsCivicLikerTrial)` ?

Reply:
The condition is Liker OR Civic Liker (Trial)

Reply:
but one cannot be a `Civic Liker (Trial)` without `getUserId()` being true?

Reply:
How about I wait for #126 to merge and I will use the new getter...

Reply:
port the getter from #126 instead? this PR seems simpler and should merge first

Reply:
Ported